src/distributing-policies/secure-supply-chain.md

```diff
@@ -0,0 +1,387 @@
+situation the users can verify the field filled with the OIDC data used during
+the signing process. For that, it's necessary to define the kind of the signature
+validation as `genericIssuer`. Therefore, it's possible to verify three piece
+of information from the signature:
+
+- `issuer`(mandatory): matches the `Issuer` attribute in the certificate
+generated by Fulcio. This shows the OIDC used to sign the policy
+- `subject`: field used to match the `Subject` attribute in the Fulcio's
+certificate. The `Subject` field contains the information of the user used to
+authenticate in the OIDC provider. The verification field, `subject`, can have
+two children fields: `equal` forces the `Subject` from the certificate to be
+equal to the value defined in the signature validation; `urlPrefix` forces the
+value of the certificate's `Subject` field value to be prefixed by the value
+defined in the signature requirement.
+
+**Examples**
+
+The following signature requirement asks the policy to be keyless signed by Alice
+using the Github OIDC:
+
+```yaml
+- kind: genericIssuer
+  issuer: https://github.com/login/oauth
+  subject:
+    equal: alice@example.com
+```
+
+This signature requirement enforces that the policy by signed in a Github actions
+from a repository owner by Flavio:
+
+```yaml
+- kind: genericIssuer
+  issuer: https://token.actions.githubusercontent.com
+  subject:
+    urlPrefix: https://github.com/flavio
+```
+
+#### Github actions signing validation
+
+This signature validation kind is used to validate policies signed in Github
+Actions. This can also be achieve with the `genericIssuer` kind as well. But
+this kind simplify the signature requirement creation by defining two fields:
+
+- `owner` (mandatory): Github ID of the user or organization to be trusted
+- `repo`: the name of the repository to be trusted
+
+**Example**
+
+```yaml
+- kind: githubAction
+  owner: kubewarden
+```
+
+#### Signature annotations validation
+
+All the signature types can have one additional optional validation field, `annotations`.
+This fields are key/value data added by the users during the signing. Thus, it is
+possible to trust policies which have some required annotations.
+
+**Examples**
+
+If the users want to trust only policies which have been signed with some key
+and the annotation `environment` with the value `production`, they can define the
+following validation:
+
+```yaml
+- kind: pubKey
+  key: |
+    -----BEGIN PUBLIC KEY-----
+    MBFKHFDGHKIJH0CAQYIKoZIzj0DAQcDQgAEX0HFTtCfTtPmkx5p1RbDE6HJSGAVD
+    BVDF6SKFSF87AASUspkQsN3FO4iyWodCy5j3o0CdIJD/KJHDJFHDFIu6sA==
+    -----END PUBLIC KEY-----
+  annotation:
+    environment: production
+```
+
+
+#### How to use signatures verification file to check a policy OCI artifact
+
+To test if a given policy passes in the signatures verification using the
+verification file, use the `--verification-config-path`  flag of the `kwctl verify` command
```

Comment:
```suggestion
verification config file, use the `--verification-config-path`  flag of the `kwctl verify` command
```

It could also be shorted to "Use `kwctl verify --verification-config-path` as shown here: (example follows)"